include/bluetooth/gatt.h

```diff
@@ -1216,7 +1216,8 @@ struct bt_gatt_exchange_params {
  *  @note Shall only be used once per connection.
  *
  *  The Response comes in callback @p params->func. The callback is run from
- *  the BT RX thread. @p params must remain valid until start of callback.
+ *  the BT RX thread, depends on `config BT_RECV_CONTEXT` selection.
```

Comment:
I prefer not mentioning the BT RX thread here as it may not be a thread.
```suggestion
 *  The Response comes in callback @p params->func. The callback is run from
 *  the context specified by 'config BT_RECV_CONTEXT'.
 *  @p params must remain valid until start of callback.
```
Similar in other occurrences.